eng/pipelines/templates/jobs/ci.yml

```diff
@@ -40,6 +40,13 @@ jobs:
       name: azsdk-pool-mms-win-2019-general
       vmImage: MMS2019
     steps:
+      - ${{ if not(contains(variables['Build.DefinitionName'], '-pr - ')) }}:
```

Comment:
Why is the -pr check necessary?

Reply:
@benbp would know. I think there was some issue with partial checkouts from private repoes. I think we need to move the check inside `sparse-checkout.yml` eventually.

Reply:
Yes I'm aware of the private repo issue but I thought it was already handled inside. I agree we should probably do that in the template.

Reply:
Yeah this can be moved into the core template I think. I can't remember if I initially added it in the java repo (as a prior example) because of time-sensitivity or a functional issue with a steplist being empty.